jina/parsers/hub/build.py

```diff
@@ -14,6 +14,8 @@ def mixin_hub_build_parser(parser):
                                            'zero or more yaml config, '
                                            'zero or more Python file. '
                                            'All files in this directory will be shipped into a Docker image')
+    gp.add_argument('-f', '--file', type=str, default='Dockerfile',
```

Comment:
I think you can make the argument of type file. Can we try that?

Reply:
the argument must be a file right? since it may come from CLI u would only be able to pass str but no file object?

Reply:
good point. I'll fix this.